packages/editor/src/components/collab-sidebar/index.js

```diff
@@ -296,19 +296,23 @@ export default function CollabSidebar() {
 			return { resultComments: [], sortedThreads: [] };
 		}
 
+		const blockCommentIds = getCommentIdsFromBlocks( blocks );
+		const blockCommentIdMap = new Map(
+			blockCommentIds.map( ( item ) => [ item.commentID, item ] )
+		);
+
 		const updatedResult = result.map( ( item ) => ( {
 			...item,
 			reply: [ ...item.reply ].reverse(),
+			clientId: blockCommentIdMap.get( item.id )?.clientID,
```

Comment:
Where is this being used? I don't see it here

Reply:
This will help to exclude the logic implemented [here](https://github.com/WordPress/gutenberg/pull/66873/files?diff=unified&w=0#diff-75adb16312a98cd771571dc3a24d08470e2ed7be595314e2f5f0176b2ba09a16R78-R97) to retrieve the `clientIds` related to `commentIds`. Also, It will help on my next PR [regarding comment board alignment to block position](https://github.com/WordPress/gutenberg/pull/67622) for the same.